packages/server/src/db/linkedRows/LinkController.ts

```diff
@@ -434,8 +433,7 @@ class LinkController {
           await this._db.put(linkedTable)
         }
       } catch (err) {
-        /* istanbul ignore next */
-        Sentry.captureException(err)
+        // Ignored error
```

Comment:
@mike12345567 @shogunpurple  should this be using posthog, or is done automatically?

Reply:
To catch errors with Posthog, we just need to console.error it, or use the `import { logging } from "@budibase/backend-core"` and `logging.logWarn` to log a message + the err object easily in a way that will be detected by Posthog.